### ISSUE TITLE
Generator Error while generating subClass

### ISSUE THREAD

**Issue by nrgjack** [madskristensen/TypeScriptDefinitionGenerator#6]
### Installed product versions
- Visual Studio: VS 2017 RC 
- This extension: 1.0.11

### Description
When generating a subclass the ts is not generated and an error is shown.

### Steps to recreate
1. Create a Person.cs class 

```
    public class Person
    {
        public string  Name { get; set; }
        public int Age { get; set; }
    }
```
2. Enable TS Generation on Person.cs
3. Create an Employee class
```
 public class Employee : Person
    {
        public decimal Salary { get; set; }
        public string Position { get; set; }
    }
```
4. (optional) build
5. Enable TS Generation on Employee.cs

### Current behavior
Person.d.ts
`
declare module server {
	interface person {
		name: string;
		age: number;
	}
}
`
then 
![image](https://cloud.githubusercontent.com/assets/11472052/23062963/63326128-f508-11e6-9683-e7fbe07ebe5b.png)
and no Employee.d.ts created

### Expected behavior
Person.d.ts
`
declare module server {
	interface person {
		name: string;
		age: number;
	}
}
`
Employee.d.ts
`declare module server {
	interface emplyee extends person {
		position: string;
		salary: number;
	}
}
`

**Comment by raboud** [comment]
Please Try Build 1.1.24 to see if your issues are corrected.

**Comment by nhammadi** [comment]
Issue resolved with the 1.1.24 release. @madskristensen you should close this issue.
